### PR TITLE
Added new token_check and JsonSuccess functions and changed HTTP responses to JSON responses

### PR DESCRIPTION
**Pull request by LaBlazer**
Changed the HttpResponses in decorators to JsonResponses for consistency, added new JsonSuccess() function which is basically just a prettier way of writing JsonResponse({}) and then added new token_check function with documentation which allows user to set up custom url for token checking.

Edit: Also changed response code on wrong credentials from 403 (Forbidden) to 401 (Unauthorized) (More info [here](https://stackoverflow.com/a/6937030))

### CODE REVIEW

File: tokenapi/decorators.py [hunk 1]
```diff
@@ -27,12 +27,12 @@ def _wrapped_view(request, *args, **kwargs):
                 user, token = auth_string.decode().split(':', 1)
 
         if not (user and token):
-            return HttpResponseForbidden("Must include 'user' and 'token' parameters with request.")
+            return JsonError("Must include 'user' and 'token' parameters with request.")
```
Comment (jpulgarin) [inline tokenapi/decorators.py]: This should be JsonResponseUnauthorized.

File: tokenapi/decorators.py [hunk 2]
```diff
@@ -27,12 +27,12 @@ def _wrapped_view(request, *args, **kwargs):
...
 
         user = authenticate(pk=user, token=token)
         if user:
             request.user = user
             return view_func(request, *args, **kwargs)
 
-        return HttpResponseForbidden()
+        return JsonResponseUnauthorized("You are unauthorized to view this page.")
```
Comment (jpulgarin) [inline tokenapi/decorators.py]: I think this should be JsonResponseForbidden.

Comment (jpulgarin) [inline tokenapi/decorators.py]: The message here should be something like, "Unable to authenticate with given 'user' and 'token' pair."

File: tokenapi/http.py [hunk 3]
```diff
@@ -19,6 +19,11 @@ def JsonResponse(data, dump=True, status=200):
         status=status,
     )
 
+def JsonSuccess():
+    data = {
+        'success': True,
+    }
+    return JsonResponse(data, status=200)
```
Comment (jpulgarin) [inline tokenapi/http.py]: The status parameter isn't needed here since JsonResponse defaults to 200.

**Comment by jpulgarin** [comment]
Thanks @LaBlazer.

I think adding the token_check view is not necessary since one can already setup a custom url using the `token` view. It would've probably been a better idea to expect the token and user as request parameter instead of view arguments from the start, but making that change now will add unnecessary complication (since we need to keep it backwards-compatible).

I like the rest of the changes though (I've left some minor comments in the diff). If you resubmit the pull request (ideally on a different branch) without the changes to README.md and tokenapi/views.py I'll merge it in.

**Comment by LaBlazer** [comment]
Thanks for the comments, I'll make a new pull request when I have some time..

**Pull request closed without merging**